13.0

mattsidnell opened the pull request:
# Fix Product dropdown inline edit on contract
Product dropdown in contracts inline edit was only showing a limited number of options even if limit was changed in module settings.  Have edited the arguments in select_produits function to show unlimited number

Commit:
new changes

mattsidnell committed:
Merge pull request #1 from Dolibarr/develop

update

mattsidnell committed:
Merge pull request #3 from Dolibarr/develop

Update

mattsidnell committed:
Merge branch 'develop' of https://github.com/mattsidnell/dolibarr into develop

mattsidnell committed:
Added ability to close contracts and made password nullable for societeaccounts

mattsidnell committed:
Merge pull request #4 from Dolibarr/develop

Catchup

mattsidnell committed:
New additions

mattsidnell committed:
Merge pull request #5 from Dolibarr/develop

get up to date

mattsidnell committed:
gg

mattsidnell committed:
Merge branch 'develop' of https://github.com/Dolibarr/dolibarr into Dolibarr-develop

mattsidnell committed:
Merge branch '13.0' of https://github.com/Dolibarr/dolibarr into 13.0

mattsidnell committed:
Fix - Contract edit inline product / service select only showing limited

eldy commented:
Something looks wrong with PR. 81 files modified ? Some changes of v13 are also lost. Did you make a merge to get last v13 branch before submitting your PR ?

mattsidnell committed:
Abandon Invoice Reason not working with ID set in formconfirm

mattsidnell committed:
Added html

mattsidnell committed:
Merge branch '13.0' of https://github.com/Dolibarr/dolibarr into 13.0

mattsidnell commented:
Hi Laurent,

I thought I did but will check it.  I have now added a new PR for the changes made and cleaned up my repository.

Matt


[Icon  Description automatically generated]
Kind regards,
Matthew Sidnell
Director
[Skype]<skype:mattsidnell>    [LinkedIn] <https://www.linkedin.com/in/matthew-sidnell/>     [WhatsApp] <https://wa.me/447876570569>
M: +44 (0) 7876 570 569
P: +44 (0) 1225 413 125
E: matt@pstructures.com<mailto:matt@pstructures.com>
[Logo]<http://pstructures.com/resources/internal/Email%20Logo.png>
[facebook icon]<https://www.facebook.com/ProfessionalStructures>    [twitter icon] <https://twitter.com/profstructures>     [linkedin icon] <https://www.linkedin.com/company/professional-structures>

www.pstructures.com<https://pstructures.com/>

The information in this email is confidential and solely for the use of the intended recipient(s). If you receive this email in error, please notify the sender and delete the email from your system immediately. In such circumstances, you must not make any use of the email or its contents. Views expressed by an individual in this email do not necessarily reflect the views of Professional Structures Limited. Computer viruses may be transmitted by email. Professional Structures Limited accepts no liability for any damage caused by any virus transmitted by this email. E-mail transmission cannot be guaranteed to be secure or error-free. It is possible that information may be intercepted, corrupted, lost, destroyed, arrive late or incomplete, or contain viruses. The sender does not accept liability for any errors or omissions in the contents of this message, which arise as a result of e-mail transmission.


From: Laurent Destailleur <notifications@github.com>
Sent: 15 January 2021 11:16
To: Dolibarr/dolibarr <dolibarr@noreply.github.com>
Cc: Matt Sidnell <matt@pstructures.com>; Author <author@noreply.github.com>
Subject: Re: [Dolibarr/dolibarr] 13.0 (#15990)


Something looks wrong with PR. 81 files modified ? Some changes of v13 are also lost. Did you make a merge to get last v13 branch before submitting your PR ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Dolibarr/dolibarr/pull/15990#issuecomment-760881275>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFCKYC2UWZ3X5OQRN3VH3XLS2APYRANCNFSM4WD2ODPQ>.
